### PR TITLE
fix: More explicit error message when DOM selector is invalid

### LINKED ISSUE #105: When we can't find a container DOM node, we should have an error that provides the selector and the widget
It would be great to have the DOM selector that we tried to use and that didn't match anything when receiving the error:

```
Error: Invariant Violation: _registerComponent(...): Target container is not a DOM element.
```

### PR DESCRIPTION
**Pull request by pixelastic**
Fixes #105

### CODE REVIEW

File: lib/utils.js [hunk 1]
```diff
@@ -1,13 +1,21 @@
 function getContainerNode(selectorOrHTMLElement) {
-  if (typeof selectorOrHTMLElement === 'string') {
-    return document.querySelector(selectorOrHTMLElement);
+  var isFromString = (typeof selectorOrHTMLElement === 'string');
```
Comment (vvo) [inline lib/utils.js]: Is there a need for parenthesis here?

**Comment by vvo** [comment]
Great, only waiting for comment on (type of)

File: lib/utils.js (hunk 1 above)
Comment (vvo) [inline lib/utils.js]: Seems they are not needed indeed

**Comment by pixelastic** [comment]
Thay might not be needed, but I feel they improve readability.

**Comment by vvo** [comment]
> They might not be needed, but I feel they improve readability.

Ok no preference here

**Comment by vvo** [comment]
LGTM

**Pull request merged**